public/gameInfo.php

```diff
@@ -1360,7 +1360,9 @@ function ResetProgress() {
                                 icon: false,
                                 tooltip: false,
                             );
-                            echo " <span class='TrueRatio'>($achTrueRatio)</span>";
+                            if ($achPoints !== 0) {
```

Comment:
this will still show "(0)" for items that have points but true points haven't been calculated yet.